test/data/manifests/arch/arch-vagrant-libvirt.mpp.json

```diff
@@ -0,0 +1,418 @@
+                  "tar",
+                  "xfsprogs",
+                  "xz",
+                  "python",
+                  "pyalpm",
+                  "grub",
+                  "mkinitcpio"
+                ]
+              }
+            }
+          },
+          "options": {}
+        }
+      ]
+    },
+    {
+      "name": "os",
+      "build": "name:build",
+      "stages": [
+        {
+          "type": "org.osbuild.pacman.conf"
+        },
+        {
+          "type": "org.osbuild.pacman",
+          "inputs": {
+            "packages": {
+              "type": "org.osbuild.files",
+              "origin": "org.osbuild.source",
+              "mpp-depsolve": {
+                "architecture": "x86_64",
+                "module-platform-id": "f34",
+                "solver": "alpm",
+                "repos": [
+                  {
+                    "id": "core",
+                    "baseurl": "https://archive.archlinux.org/repos/2021/11/24/$repo/os/$arch"
+                  },
+                  {
+                    "id": "community",
+                    "baseurl": "https://archive.archlinux.org/repos/2021/11/24/$repo/os/$arch"
+                  },
+                  {
+                    "id": "extra",
+                    "baseurl": "https://archive.archlinux.org/repos/2021/11/24/$repo/os/$arch"
+                  }
+                ],
+                "packages": [
+                  "base",
+                  "bash",
+                  "pacman",
+                  "btrfs-progs",
+                  "dosfstools",
+                  "e2fsprogs",
+                  "systemd",
+                  "linux",
+                  "mkinitcpio",
+                  "tar",
+                  "grub",
+                  "openssh",
+                  "virtualbox-guest-utils-nox",
+                  "polkit",
+                  "networkmanager"
+                ]
+              }
+            }
+          },
+          "options": {}
+        },
+        {
+          "type": "org.osbuild.systemd",
+          "options": {
+            "enabled_services": [
+              "vboxservice.service",
+              "sshd.service",
+              "NetworkManager.service"
+            ]
+          }
+        },
+        {
+          "type": "org.osbuild.users",
+          "options": {
+            "users": {
+            "vagrant": {
+              "password": "$6$8dbFyte9oE3ugPHO$q0cTMv1oAgK/ZVXUTSUqj1aXzrhniNqfylqbcW.LgElYRJNRGSr4hBE7hghu2oP1nKn68u13/YmDkKH.s6yil0",
+              "home": "/home/vagrant",
+              "key": "ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEA6NF8iallvQVp22WDkTkyrtvp9eWW6A8YVr+kz4TjGYe7gHzIw+niNltGEFHzD8+v1I2YJ6oXevct1YeS0o9HZyN1Q9qgCgzUFtdOKLv6IedplqoPkcmF0aYet2PkEDo3MlTBckFXPITAMzF8dJSIFo9D8HfdOV0IAdx4O7PtixWKn5y2hMNG0zQPyUecp4pzC6kivAIhyfHilFR61RGL+GPXQ2MWZWFYbAGjyiYJnAmCP3NOTd0jMZEnDkbUvxhMmBYSdETk1rRgm+R4LOzFUGaHqHDLKLX+FIPKcF96hrucXzcWyLbIbEgE98OHlnVYCzRdK8jlqm8tehUc9c9WhQ== vagrant insecure public key"
```

Comment:
This could be an input, as the key is from https://github.com/hashicorp/vagrant/raw/main/keys/vagrant.pub which can change in the future so not hardcoding would be nice!

Reply:
For the rpm stage we just embed the GPG keys very much like you have done here. The input reference is the content hash so it would change if the key changes. Might still be nicer. Not sure.